cli/lsp/language_server.rs

```diff
@@ -3242,6 +3243,7 @@ impl tower_lsp::LanguageServer for LanguageServer {
 
     {
       let mut ls = self.0.write().await;
+      init_log_file(ls.config.persist_log());
```

Comment:
Does this require restarting the LSP on change? Seems like it would be somewhat easier to now have this done whenever the config changes?

Reply:
I think the file shouldn't change mid-process or it could be hard to debug something going wrong with that for example. The mechanism itself should be impervious to runtime stuff imo

Reply:
And we can add it as a restart-when-changed setting: https://github.com/denoland/vscode_deno/blob/3.28.0/client/src/extension.ts#L175-L184